package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "react-components",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "format:fix": "prettier . --write",
+    "prepare": "husky install",
+    "pre-commit": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0 && git add ."
```

Comment:
Please, clarify what behavior are you trying to achieve adding ```git add .``` to the end of script?